www/resources/views/pages/testhistory.blade.php

```diff
@@ -35,34 +35,31 @@
         <div class="history-controls">
             <input id="CompareBtn" type="submit" value="Compare Selected Tests">
         </div>
-        <div class="scrollableTable">
-            <table id="history" class="history pretty" border="0" cellpadding="5px" cellspacing="0">
-                <thead>
-                    <tr>
-                        <th class="pin"><span>Select to compare</span></th>
-                        <th class="url">URL</th>
-                        <th class="date">Run Date</th>
-                        <th class="location">Run From</th>
-                        <th class="label">Label</th>
-                    </tr>
-                </thead>
-                @if ($is_logged_in)
-                    <tbody id="historyBody">
-                        @foreach ($test_history as $record)
-                            <tr>
-                                <th><input type="checkbox" name="t[]" value="{{ $record->getTestId() }}" aria-label="Select this test" /></th>
-                                <td class="url"><a href="/result/{{ $record->getTestId() }}/">{{ $record->getUrl() }}</a></td>
-                                <td class="date">{{ date_format(date_create($record->getStartTime()), 'M d, Y g:i:s A e') }}</td>
-                                <td class="location">{{ $record->getLocation() }}</td>
-                                <td class="label">{{ $record->getLabel() }}</td>
-                            </tr>
-                        @endforeach
-                    </tbody>
-                @endif
-            </table>
-        </div>
-
-
+        <table id="history" class="history pretty" border="0" cellpadding="5px" cellspacing="0">
+            <thead>
+                <tr>
+                    <th><span>Select to compare</span></th>
```

Comment:
what's reason to remove pin class?

Reply:
@v-hclaveria  bump on this

Reply:
The pin class has the property border-left: 0, disappearing the left line of the box
![image](https://github.com/catchpoint/WebPageTest/assets/149002067/1a72beac-51b2-471e-96c2-84f670322337)